chcemvediet/tests/__init__.py

```diff
@@ -48,6 +49,21 @@ def run_tests(self, *args, **kwargs):
 
 class ChcemvedietTestCaseMixin(TestCase):
 
+    def __init__(self, methodName):
+        super(ChcemvedietTestCaseMixin, self).__init__(methodName)
+        self.patterns_single_column = [
+            u'FROM "accounts_profile" WHERE "accounts_profile"."user_id" = %s LIMIT 21',
+            squeeze(u"""
```

Comment:
@viliambalaz Nazov `patterns_single_column` je trochu matuci, kedze tato query sa nevykona len v `main/base/single_column.html`, ale vo vsetkych templatoch, co pouzivaju `main/base/base.html`. Tam sa to vykona v headri.

Takze by som to skor nazval `patterns_base`

Reply:
:ok: premenovane, nazval som to preto tak, lebo tieto patterny vznikli prave v template `main/base/single_column.html`